I Need HELP with assignment 5 in week 2.!!!!!!!!!!!!!!!

wholewheattoast opened the pull request:
Hey Tony, this isn't working.

When I added FOO to line 3 it all broke... help!

wholewheattoast committed:
I Need HELP with assignment 5 in week 2.!

Hey Tony, this isn't working.

When I added FOO to line 3 it all broke... help!

programmist commented on week-2/5/ex-5.html:
```diff
@@ -0,0 +1,23 @@
+      </div>
+    );
+
+    something bad here
```
You forgot HTML tags

wholewheattoast replied:
Hey @programmist i fixed that thing, is it better?

wholewheattoast committed:
Yay, fix syntax error in html tag

Resolving Pull Request feedback at https://github.com/programmist/react-course-1/pull/1/files#r401877287

programmist reviewed (approved):
LGTM
